### ISSUE TITLE
RealmSwift cocoapod for Xcode 8 and Swift 3?

### ISSUE THREAD

**Issue by awclives** [realm/realm-cocoa#4097]
Is there one yet?  I have all my other dependencies updated . . . except for your excellent and important one.

Thanks.  Love your software.

**Comment by rjvir** [comment]
+1

**Comment by jpsim** [comment]
We've maintained support for Realm Swift across Xcode 8 betas since WWDC, making adjustments to continue compiling and passing tests for Swift 2.2, 2.3 and 3.0 as new betas came out.

However, due to the amount of churn Swift 3 has seen in the last 3 months, we held off on thoroughly reviewing all our Swift 3 API changes and accommodating both Xcode 7.3.1 and Xcode 8 into our continuous integration, testing, docs and release packaging processes.

With Xcode 8 GM last week, we decided to [make a push](https://twitter.com/realm/status/773665903946952704) to finalize our Swift 3 APIs along with the other integration points mentioned above.

That work is now mostly complete and we're preparing a 1.1.0 release for tomorrow, at which point our Swift 3 APIs will be final, binaries will be made available for Swift 2.3 and 3.0 in our release zips, and docs will be updated to reflect the new APIs.

Thanks for your patience.
JP
